lib/ruby-debug-ide/xml_printer.rb

```diff
@@ -1,8 +1,8 @@
 require 'stringio'
 require 'cgi'
 require 'monitor'
-if (!defined?(JRUBY_VERSION))
-  include ObjectSpace
+if (!defined?(JRUBY_VERSION) && RUBY_VERSION >= '2.0')
```

Comment:
What will `ObjectSpace.memsize_of_all` do if `RUBY_VERSION < "2.0"`?

Reply:
(-Inf; 1.9.3) -> no method exception, [1.9.3; 2.0) -> does what it should, but we have no tracepoint so it is useless

Reply:
So that file does not work with Rubies < 2.0? `exec_with_allocation_control` should not work with them in that case.